utils/appliance/__init__.py

```diff
@@ -142,6 +148,36 @@ def uninstall_ipa_client(self):
         assert return_code != 0
 
 
+class ApplianceLoggerAdapter(logging.LoggerAdapter):
```

Comment:
I wonder - can we extend the data passed through the adaptors? Eg we would have `tags`, appliance would add `appliance`, ssh client would add `ssh` et cetera - so once we start processing the logs in some sort of elk/splunk, we can have all the tags from all the adaptors the message passed through? @RonnyPfannschmidt